src/clang.rs

```diff
@@ -1409,7 +1409,7 @@ impl Drop for Diagnostic {
 /// A file which has not been saved to disk.
 pub struct UnsavedFile {
     x: CXUnsavedFile,
-    name: CString,
+    pub name: CString,
```

Comment:
CI is failing because this field is public but it's not documented. Please add a line like:

```
/// The name of the unsaved file. We carry that over to avoid leaving dangling pointers in `CXUnsavedFile`
```

Or something like that.